python/pyspark/ml/regression.py

```diff
@@ -502,7 +526,9 @@ def getFeatureIndex(self):
         return self.getOrDefault(self.featureIndex)
 
 
-class IsotonicRegressionModel(JavaModel, JavaMLWritable, JavaMLReadable):
+class IsotonicRegressionModel(JavaModel, JavaMLWritable, JavaMLReadable,
+                              HasFeaturesCol, HasLabelCol, HasPredictionCol,
+                              HasWeightCol):
```

Comment:
nit: keep ordering of inheritance consistent - other places put `JavaMLWritable, JavaMLReadable` last